ios/FluentUI.Demo/FluentUI.Demo/ReadmeViewController.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+import FluentUI
+
+class ReadmeViewController: UIViewController {
+
+    private struct Constants {
+        static let topPadding: CGFloat = 25
+        static let leadingAndTrailingPadding: CGFloat = 20
+        static let popoverWidth: CGFloat = 400
+    }
+
+    private let readmeLabel: Label = {
+        let label = Label()
+        label.numberOfLines = 0
+
+        // TODO: Change color to fluent 2 tokens
+        label.textColor = UIColor(dynamicColor: DynamicColor(light: ColorValue(0x000000),
+                                                             dark: ColorValue(0xFFFFFF)))
+        label.translatesAutoresizingMaskIntoConstraints = false
+        return label
+    }()
+
+    private let readmeString: String?
+
+    override func viewDidLayoutSubviews() {
+        super.viewDidLayoutSubviews()
+
+        if let readmeString = readmeString {
+            readmeLabel.text = readmeString
+        }
+
+        setupConstraints()
```

Comment:
everything about setupconstraint to setting label seems like we should do in load view

Reply:
Yep, the constratints logic is moved to viewDidLoad